go.mod

```diff
@@ -1,8 +1,12 @@
 module github.com/kyoh86/go-spdx
```

Comment:
it's unnecessary. 
run `go mod tidy` plz.

Reply:
@kyoh86 done